insights/2015-04-01/swagger/insightsClient_TenantEvents.json

```diff
@@ -0,0 +1,297 @@
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "the locale specific value."
+        }
+      },
+      "description": "The localizable string class."
+    },
+    "SenderAuthorization": {
+      "properties": {
+        "action": {
+          "type": "string",
+          "description": "the permissible actions. For instance: microsoft.support/supporttickets/write"
+        },
+        "condition": {
+          "type": "string",
+          "description": "the condition."
+        },
+        "role": {
+          "type": "string",
+          "description": "the role of the user. For instance: Subscription Admin"
+        },
+        "scope": {
+          "type": "string",
+          "description": "the scope."
+        }
+      },
+      "description": "The RBAC properties of the event."
+    },
+    "HttpRequestInfo": {
+      "properties": {
+        "clientRequestId": {
+          "type": "string",
+          "description": "the client request id."
+        },
+        "clientIpAddress": {
+          "type": "string",
+          "description": "the client Ip Address"
+        },
+        "method": {
+          "type": "string",
+          "description": "the Http request method."
+        },
+        "uri": {
+          "type": "string",
+          "description": "the Uri."
+        }
+      },
+      "description": "The Http request info."
+    },
+    "EventData": {
+      "required": [ "channels", "level", "eventTimestamp", "submissionTimestamp" ],
+      "properties": {
+        "authorization": {
+          "$ref": "#/definitions/SenderAuthorization",
+          "description": "the authorization used by the user who has performed the operation that led to this event. This captures the RBAC properties of the event. These usually include the 'action', 'role' and the 'scope'"
+        },
+        "channels": {
+          "type": "string",
+          "description": "the event channels. The regular event logs, that you see in the Azure Management Portals, flow through the 'Operation' channel.",
+          "enum": [
+            "Admin",
+            "Operation",
+            "Debug",
+            "Analytics"
+          ],
+          "x-ms-enum": {
+            "name": "EventChannels",
+            "modelAsString": false
+          }
+        },
+        "claims": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "key value pairs to identify ARM permissions."
+        },
+        "caller": {
+          "type": "string",
+          "description": "the email address of the user who has performed the operation, the UPN claim or SPN claim based on availability."
+        },
+        "description": {
+          "type": "string",
+          "description": "the description of the event."
+        },
+        "id": {
+          "type": "string",
+          "description": "the resource Id."
```

Comment:
Can you provide more meaningful information over here? The resource id for which resource? 
Is this the resource Id of the event or the resource id of the resource corresponding to the event?